js/siteorigin-panels/dialog/widget.js

```diff
@@ -156,12 +156,12 @@ module.exports = panels.view.dialog.extend( {
         if( !this.model.get('missing') ) {
             // Only get the values for non missing widgets.
             var values = this.getFormValues();
-            if (typeof values.widgets === 'undefined') {
+            if ( typeof values.widgets === 'undefined' ) {
```

Comment:
Not a problem, just a question. Are we following [jQuery style guide](https://contribute.jquery.org/style-guide/js/) conventions? We should probably decide which convention we're using and make a note of it somewhere in our docs or readme's.